Amplify/Categories/Auth/AuthCategoryBehavior.swift

```diff
@@ -71,6 +71,8 @@ public protocol AuthCategoryBehavior {
 
     /// SignIn using pre configured web UI.
     ///
+    /// Calling this method will always launch a webUI that let the user to signIn.
```

Comment:
reword: "Calling this method will always launch the Auth plugin's default web user interface"